test/support/pages/header_page.js

```diff
@@ -56,8 +60,12 @@ export default (function () {
     },
 
     clickAbsoluteButton: function clickAbsoluteButton() {
+      var self = this;
       return this.remote.setFindTimeout(defaultFindTimeout)
-      .findByLinkText('Absolute').click();
+      .findByLinkText('Absolute').click()
+      .then(function () {
+        return self.getSpinnerDone();
```

Comment:
this shouldn't be needed should it?  clicking absolute shouldn't be triggering any requests